python/test_tools/s3control_stubber.py

```diff
@@ -0,0 +1,92 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Stub functions that are used by the AWS S3 Control unit tests.
```

Comment:
by the **Amazon** S3

Reply:
It turns out there are two APIs: "Amazon S3" and "AWS S3 Control", so keeping AWS here.